static/src/main.ts

```diff
@@ -24,4 +30,36 @@ app.use(Toaster);
 
 app.use(VueCookies, { expire: "1d" });
 
+const routes = [
+  { path: "/", component: RedirectView },
+  {
+    path: "/login",
+    component: LoginView,
+    beforeEnter: () => {
+      return store.isLoggedIn ? "/tasks" : true;
```

Comment:
Do I understand correctly that this directly redirects me to the task-overview if I'm already logged in and if not just continues to send me to the login page?